### PR TITLE
Added Olivier’s Theme

### PR DESCRIPTION
**Pull request by OlivierPS**
# I am submitting a new Community Theme

## Repo URL

Link to my theme: https://github.com/OlivierPS/Olivier-s-Theme

## Theme checklist

- [x] My repo contains all required files (please *do not* add them to this `obsidian-releases` repo).
  - [x] `manifest.json`
  - [x] `theme.css`
  - [x] The screenshot file (16:9 aspect ratio, recommended size is 512px by 288px for fast loading).
- [x] I have indicated which modes (dark, light, or both) are compatible with my theme.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other themes that I'm using. I have given proper attribution to these other themes in my `README.md`.

### CODE REVIEW

**Comment by OlivierPS** [comment]
A user guide is linked to in the ReadMe file.

**Comment by joethei** [comment]
- your theme screenshot is too big, recommended size is 512px by 288px

File: community-css-themes.json [hunk 1]
```diff
@@ -1346,5 +1346,13 @@
...
         "screenshot": "img/sparkling_night.png",
         "modes": ["dark", "light"]
+    },
+     {
+        "name": "Olivier’s Theme",
+        "author": "Olivier Spinnler",
+        "repo": "OlivierPS/Olivier-s-Theme",
+        "screenshot": "main-screenshot.png",
+        "modes": ["dark", "light"],
+        "branch": "master"
```
Comment (joethei) [inline community-css-themes.json:1356]: remove this `branch` parameter, it's no longer used.

**Comment by OlivierPS** [comment]
- [x] screenshot corrected
- [x] suppressed the "branch" parameter — I hope  I did it the correct way

**Comment by OlivierPS** [comment]
I hope what I did is correct.

File: community-css-themes.json (hunk 1 above)
Comment (OlivierPS) [reply community-css-themes.json]: Done !

**Pull request merged**